### PR TITLE
UG - Fix Quick Start section and broken link to releases

### LINKED ISSUE #145: [PE-D][Tester E] Quick Start guide is not very specific
More details ie what command to enter into terminal/ CommandPrompt to start the app

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: jorellesee/ped#1

### LINKED ISSUE #182: [PE-D][Tester B] Broken link of app download in UG
As of 1900 hours, 28 Oct, the link of "latest version" directs to http://link.to/duke, an unresolvable website.

This issue is marked with medium severity as it is important a new user who do not know other channels of downloading your jar file. Please remember to update all the links, and double-check on these important links. 

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: HT-T/ped#9

### PR DESCRIPTION
**Pull request by ngdeqi**
Fixes #182 
Fixes #145

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -29,7 +29,17 @@ information quickly with minimal latency.
 ## Quick Start
 
 1. Ensure that you have Java 11 or above installed. 
-2. Download the latest version of `Property Rental Manager` from [here](http://link.to/duke).
+2. Download the latest version of `PropertyRentalManager.jar` from [here](https://github.com/AY2223S1-CS2113-F11-1/tp/releases).
```
Comment (FeliciaBeatrice) [inline docs/UserGuide.md:32]: everything looks good! steps are easy to understand :)

Comment (wilsonngja) [reply docs/UserGuide.md:32]: I agree. Thanks for also fixing the broken link.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -29,7 +29,17 @@ information quickly with minimal latency.
...
+3. Put the JAR file into an empty folder.
+4. Open a command window and change the current working directory to the directory that the JAR file is located in using
+   following command:
+```
+cd [PATH_TO_JAR_DIRECTORY]
```
Comment (wilsonngja) [inline docs/UserGuide.md:37]: Could consider what does each command do?
For example, changing to the directory or running the JAR file.

**Review by wilsonngja — APPROVED** [review]
Good job in fixing the errors. LGTM.

**Review by wilsonngja — APPROVED** [review]
Good job in fixing the errors. Everything LGTM.

**Pull request merged**